packages/hash/external-services/kratos/kratos.prod.yml

```diff
@@ -23,18 +23,17 @@ selfservice:
     password:
       enabled: true
 
-    # Email sending diabled in production for now
-    # link:
-    #   config:
-    #     # The URL for verification emails are set through the link method
-    #     # but we're using the code method, so we disable this method for usage.
-    #     enabled: false
-    #     # Set through SELFSERVICE_METHODS_LINK_CONFIG_BASE_URL
-    #     base_url: http://localhost:3000/api/ory
-    # code:
-    #   config:
-    #     # and make sure to enable the code method.
-    #     enabled: true
+    link:
+      config:
+        # The URL for verification emails are set through the link method
+        # but we're using the code method, so we disable this method for usage.
+        enabled: false
+        # Set through SELFSERVICE_METHODS_LINK_CONFIG_BASE_URL
+        base_url: http://localhost:3000/api/ory
```

Comment:
Can we remove this if we're disabling this method?
```suggestion
```

I'm also a bit unsure why we would have `http://localhost:3000` in the production file in any case.

Reply:
Because of some strageness in the way the email templates are configured, the base url of the link method is used for the code flow emails